sharktank/tests/types/dataset_test.py

```diff
@@ -143,6 +144,61 @@ def testDatasetRoundtrip(self):
             "a.c.d", ExternalTensorTrait.get(t_acd.as_torch()).external_name
         )
 
+    def testRoundTripDefaultPrimitiveTensor(self):
+        t_orig = DefaultPrimitiveTensor(
+            name="primitive", data=torch.Tensor([1.0, 2.0, 3.0])
+        )
+        ds_orig = Dataset({}, Theta({t_orig.name: t_orig}))
+        ds_orig.save(self.temp_dir / "p_ds.irpa")
+
+        ds_load = Dataset.load(self.temp_dir / "p_ds.irpa", mmap=False)
+        t_load = ds_load.root_theta.tensor(t_orig.name)
+        print(t_load)
```

Comment:
Here and on other places better to log with a level to not pollute the console aways.

Reply:
Done